streams/src/main/java/org/apache/kafka/streams/processor/internals/metrics/StreamsMetricsImpl.java

```diff
@@ -453,12 +469,59 @@ public static void addInvocationRateAndCount(final Sensor sensor,
                                                  final String group,
                                                  final Map<String, String> tags,
                                                  final String operation) {
-        addInvocationRateAndCount(sensor,
-                                  group,
-                                  tags,
-                                  operation,
-                                  "The total number of " + operation,
-                                  "The average per-second number of " + operation);
+        addInvocationRateAndCount(
+            sensor,
+            group,
+            tags,
+            operation,
+            "The total number of " + operation,
+            "The average per-second number of " + operation
+        );
+    }
+
+    public static void addAmountRateAndTotalMetricsToSensor(final Sensor sensor,
+                                                            final String group,
+                                                            final Map<String, String> tags,
+                                                            final String operation,
+                                                            final String descriptionOfRate,
+                                                            final String descriptionOfTotal) {
+        addAmountRateMetricToSensor(sensor, group, tags, operation, descriptionOfRate);
+        addTotalMetricToSensor(sensor, group, tags, operation, descriptionOfTotal);
+    }
+
+    public static void addAmountRateMetricToSensor(final Sensor sensor,
```

Comment:
To make the function name consistent with the Stat name, should this be `addRateOfSum`?

Reply:
> This is not for this PR, but: since in #7057 we are cleaning up the confusing names shall we do the same for function names as well as a follow-up? Also the existing function names do not have `..ToSensor` suffix, could we make it consistent (I'm fine with either way, just want to do the same for new and existing functions)?

I agree with you that we should be consistent. I just did not want to make this PR bigger than needed since it is already quite big. I will open a refactoring PR for this.